### ISSUE TITLE
How to set weight to each binary classification of multiple catogories?

### ISSUE THREAD

**Issue by wenweixu** [keras-team/keras#9316]
I have 16 categories, each of which is a binary classification, because the categories are not mutually exclusive.
I am using binary_crossentropy as loss function. I hope to put more weight on the 1s in each of the category. How to achieve that?
Right now the class_weight option seems to only allow me to set weight for each category instead of each class of each category. for example: class_weight  = {0:1, 1:1, 2:1,3:1, 4:1, 5:1, 6:1, 7:1, 8:1, 9:1, 10:1, 11:1, 12:1, 13:1, 14:1, 15:1} will set weight to each of the 16 catogories, which is not what I want.
This is my first question post on Github. Thanks for help.

**Comment by brge17** [comment]
Either rewrite the class weights to do what you want. Or use instance level weights  (sample weights) with a generator.
